src/index.ts

```diff
@@ -61,6 +61,12 @@ export interface CheckoutOptions {
   testing?: boolean
   /** Wether the Checkout should be a popup or an inline element  */
   display?: string
+  /** Wether the Checkout should have the customer details form hidden */
+  hideDetails?: boolean
+  /** Which language should the Checkout be displayed */
```

Comment:
Still missing the `in` somewhere in the sentence, like `Which language should the Checkout be displayed in` or `The language in which the Checkout should be displayed`, if you prefer.